.github/workflows/ci.yml

```diff
@@ -27,7 +27,30 @@ jobs:
       with:
         name: linux-test-reports
         path: build/reports
-      if: success() || failure()
+      if: always()
+    - name: Run integration tests
+      run: |
+        dir=~/.java/.userPrefs/jetbrains/_\!\(\!\!cg\"p\!\(\}\!\}@\"j\!\(k\!\|w\"w\!\'8\!b\!\"p\!\':\!e@=\=
+        mkdir -p "$dir"
+        cp .github/workflows/prefs.xml $dir
+        mkdir -p ~/.local/share/JetBrains/consentOptions
+        cp .github/workflows/accepted ~/.local/share/JetBrains/consentOptions
+        export DISPLAY=:99.0
+        Xvfb -ac :99 -screen 0 1920x1080x16 &
+        ./gradlew runIdeForUiTests -PideaVersion=2020.1 &
```

Comment:
Shouldn't we target latest IJ version ie 2020.3 ?

Reply:
Yes but the IDE has very different UI from 2020.1 (which is currently supported by the UI tests). I will perform updates of UI tests very soon in separated PR, I have jira for that already created - https://github.com/redhat-developer/intellij-quarkus/issues/317 (I do not want to make this PR too big and keep it stucked too long before merging).